packages/editor/src/managers/clipboard/clip-event-dispatch.ts

```diff
@@ -54,19 +54,27 @@ export class ClipEventDispatch {
     });
   }
 
+  static path(event: ClipboardEvent): EventTarget[] {
+    if ('path' in event) {
+      // @ts-ignore
+      return event.path;
```

Comment:
Can we use `event.path as unknown as EventTarget[]` to remove the `// @ts-ignore`?

Reply:
no